### ISSUE TITLE
mp4v2

### ISSUE THREAD

**Issue by exekutive** [sandreas/m4b-tool#256]
is there a reason you have a custom mp4v2 tap, or does the default homebrew formula work exactly the same?

when I get info on yours, HB says:

```
https://github.com/sandreas/mp4v2
Conflicts with:
  bento4 (because both install `mp4extract` and `mp4info` binaries)

```

**Comment by sandreas** [comment]
@exekutive You are right - the tap is more or less deprecated. However, the custom tap is tested and nothing too exciting changed in enzos branch.

I try to update the dependencies and the documentation, but for now I strongly recommend to use `docker` (the official image) in favour of brew.

**Comment by exekutive** [comment]
I'd rather not install extra crap just for one library. But thanks.
